apps/storybook/stories/Namekit/RegistrationPriceInfo.stories.tsx

```diff
@@ -1,33 +1,39 @@
 import {
   PrimaryRegistrationStatus,
   SecondaryRegistrationStatus,
+  buildTimestamp,
+  now,
 } from "@namehash/ens-utils";
 import type { Meta, StoryObj } from "@storybook/react";
 import {
   RegistrationPriceInfo,
   RegistrationPriceInfoDisplayingFormat,
 } from "@namehash/namekit-react";
 
+const ELEVEN_DAYS_FROM_NOW_TIMESTAMP = buildTimestamp(
+  BigInt(new Date().getTime()) / 1005n,
+);
+
 export const DisplayIconWithTooltip: Story = {
   args: {
     displayIconWithTooltip:
       RegistrationPriceInfoDisplayingFormat.IconWithTooltip,
     registration: {
-      expiryTimestamp: { time: 1709652875n },
+      expiryTimestamp: ELEVEN_DAYS_FROM_NOW_TIMESTAMP,
       primaryStatus: PrimaryRegistrationStatus.Expired,
       secondaryStatus: SecondaryRegistrationStatus.RecentlyReleased,
-      registrationTimestamp: { time: 1711380875n },
-      expirationTimestamp: { time: 1709652875n },
+      registrationTimestamp: now(),
+      expirationTimestamp: now(),
     },
     parsedName: {
       namehash:
-        "0xc4feb2ef28ae93ad2847cc7aaa8d614509f29feea35c86555550f071e26cec83",
-      slug: "xtest$1709652875$$$false.eth",
-      displayName: "xtest$1709652875$$$false.eth",
-      normalizedName: "xtest$1709652875$$$false.eth",
-      labelName: "xtest$1709652875$$$false",
+        "0x54f4b08bc10ab9368020f2afdd830ab742058481586c4ae87ba6e37c8912a6ec",
+      slug: "expiredDomain.eth",
```

Comment:
A lot of the values in this file are `expiredDomain.eth` with an uppercase D which is unnormalized. Can you please only use normalized name examples in this storybook? Please also update namehash / etc.. values as needed for this change too.